package-lock.json

```diff
@@ -19930,4 +19930,4 @@
       }
     }
   }
-}
+}
```

Comment:
why make this change?

Reply:
I took the package.lock.json file present in developer branch and replaced that with the one present in my branch

Reply:
Line ending seems to be changed.

@bodhish should we add `.gitattributes` to always use LF line endings? https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#per-repository-settings